### ISSUE TITLE
unified memory

### ISSUE THREAD

**Issue by MarcusGDaniels** [heavyai/heavydb#635]
Is it possible/sensible to use unified memory to avoid errors like:

2021-03-22T19:54:56.782153 E 118135 617 BufferMgr.cpp:410 ALLOCATION failed to find 256000000B throwing out of memory GPU_MGR:3
2021-03-22T19:54:57.056019 E 118135 614 BufferMgr.cpp:410 ALLOCATION failed to find 256000000B throwing out of memory GPU_MGR:0
2021-03-22T19:54:57.314486 E 118135 616 BufferMgr.cpp:410 ALLOCATION failed to find 256000000B throwing out of memory GPU_MGR:2
2021-03-22T19:54:57.705913 E 118135 615 BufferMgr.cpp:410 ALLOCATION failed to find 256000000B throwing out of memory GPU_MGR:1
2021-03-22T19:54:57.714073 E 118135 1 DBHandler.cpp:1091 Exception: Failed to run the cardinality estimation query: Query couldn't keep the entire working set of columns in GPU memory

**Comment by cdessanti** [comment]
Nope, the two memory pools are managed internally and the way the system memory is allocated isn't compatible with unified memory. 

I could instruct you how to set database parameters to stream data from system memory to GPU, but it wouldn't be a supported configuration and, likely, the performance wouldn't be optimal

**Comment by psilvest** [comment]
This issue has been inactive for a while, and the project has changed substantially. If this still occurs on the latest release, please add reproduction steps and environment details.
